.github/workflows/draft-release.yml

```
@@ -0,0 +1,35 @@
+name: draft-release
+
+on:
+  pull_request: ~
```

[req] needs to be removed before merge!!!
```suggestion
  pull_request: ~
```